testFramework/com/twitter/intellij/pants/testFramework/OSSPantsIntegrationTest.java

```diff
@@ -80,4 +84,21 @@ protected void assertNotContainsSubstring(String s, String unexpected) {
   protected void assertEmptyBeforeRunTask(RunConfiguration configuration) {
     assertEmpty(getBeforeRunTask(configuration));
   }
+
+  /**
+   * Assert Project has the right JDK and language level (JVM project only).
+   */
+  protected void assertProjectJdkAndLanguageLevel() {
+    assertEquals(
+      ProjectRootManager.getInstance(myProject).getProjectSdk().getHomePath(),
+      PantsUtil.getDefaultJavaSdk(getParentPath()).get().getHomePath()
+    );
+
+    LanguageLevel projectLanguageLevel = LanguageLevelProjectExtension.getInstance(myProject).getLanguageLevel();
+    LanguageLevel expectedLanguageLevel = LanguageLevel.JDK_1_8;
```

Comment:
This requires my laptop and travis to have different java version installed, which is a bit hard to accommodate. Since I did not explicitly set language level anywhere and the jdk level was determined via IntelliJ framework's library, I'd imagine the existing test should suffice.